config/proto/probe.proto

```diff
@@ -1,9 +1,22 @@
-/*
-  This proto defines the initial probe interface for Hermes probes.
-  It is used in a variety of other files as it is the top-level probe interface.
-  
-  Author: Evan Spendlove (@evanSpendlove)
-*/
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author: Evan Spendlove (@evanSpendlove)
+//
+//  This proto defines the probe interface for Hermes probes.
```

Comment:
According to the protobuf style guide, repeated fields should have prularized names.
It's confusing that this is named using plurals, but it's not a repeated field.
I would just replace this with a repeated field.

Reply:
I am addressing comments on the .proto files in a different PR, as this branch is meant to only include the Cloudprober integration files. These .proto files are the same as those in the master branch.

Please see the new PR for the branch: evan/protoChanges.

-- 

I have made this a repeated field as requested.